purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEvent.kt

```diff
@@ -38,18 +38,43 @@ class CustomerCenterEvent(
     )
 
     @ExperimentalPreviewRevenueCatPurchasesAPI
-    @Poko
     @Serializable
+    @Poko
     @SuppressWarnings("LongParameterList")
     class Data(
         val type: CustomerCenterEventType,
         @Serializable(with = DateSerializer::class)
         val timestamp: Date,
+        @Serializable(with = UUIDSerializer::class)
+        val sessionIdentifier: UUID,
         val darkMode: Boolean,
         val locale: String,
         val isSandbox: Boolean,
         val version: Int = 1,
-        val revisionId: Int = 1,
+        val revisionID: Int = 1,
         val displayMode: CustomerCenterDisplayMode = CustomerCenterDisplayMode.FULL_SCREEN,
-    )
+    ) {
+        @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+        fun copy(
```

Comment:
We could make this `internal` I think? Since it's only used for tests

Reply:
```suggestion
        internal fun copy(
```